docs/applications/implementations/models.md

```diff
@@ -50,18 +50,50 @@ def create_estimator(run_config, model_config):
 
 You can import PyPI packages or your own Python packages to help create more complex models. See [Python Packages](../advanced/python-packages.md) for more details.
 
+The following packages have been pre-installed and can be used in your implementations:
+
+```text
+tensorflow==1.12.0
+boto3==1.9.78
+msgpack==0.6.1
+numpy>=1.13.3,<2
+requirements-parser==0.2.0
+packaging==19.0.0
+```
+
+You can install additional PyPI packages and import your own Python packages. See [Python Packages](../advanced/python-packages.md) for more details.
+
 
 # Tensorflow Transformations
-You can preprocess input features and labels to your model by defining a `transform_tensorflow` function. This is useful in cases where you don't have access to the model implementation which expects a multi-dimensional tensor, for example.
+You can preprocess input features and labels to your model by defining a `transform_tensorflow` function. An example of when this might be useful is to reshape a tensor to feed into a pre-made model.
 
 ```python
 def transform_tensorflow(features, labels, model_config):
+     """Create a mapping function for the tf.Dataset API. You can define
+     tensor transformations you want to apply to the features and labels of
+     each training sample.
+
+    Args:
+        features: A feature dictionary of column names to feature tensors.
+
+        labels: The label tensor.
+
+        model_config: The Cortex configuration for the model.
+            Note: nested resources are expanded (e.g. model_config["target_column"])
+            will be the configuration for the target column, rather than the
+            name of the target column).
+
+
+    Returns:
+        features and label tensors for the current sample.
+    """
+
     hparams = model_config["hparams"]
```

Comment:
Maybe we should write `pass` here, and add a subsection for `Example` to make it clear that this is an example? (similar to above) (feel free to ask Omer)